src/olympia/reviewers/templates/reviewers/review.html

```diff
@@ -319,20 +319,22 @@ <h3 id="history">
                   data-toggle-button-selector="#force_disable_addon"
                   id="force_enable_addon" type="button">{{ _('Force-enable add-on') }}</button>
         </li>
-        <li {% if addon.auto_approval_disabled %}class="hidden"{% endif %}>
-          <button data-api-url="{{ url('reviewers-addon-flags', addon.pk) }}"
-                  data-api-method="patch"
-                  data-api-data="{&quot;auto_approval_disabled&quot;: true}"
-                  data-toggle-button-selector="#enable_auto_approval"
-                  id="disable_auto_approval" type="button">{{ _('Disable Auto-Approval') }}</button>
-        </li>
-        <li {% if not addon.auto_approval_disabled %}class="hidden"{% endif %}>
-          <button data-api-url="{{ url('reviewers-addon-flags', addon.pk) }}"
-                  data-api-method="patch"
-                  data-api-data="{&quot;auto_approval_disabled&quot;: false}"
-                  data-toggle-button-selector="#disable_auto_approval"
-                  id="enable_auto_approval" type="button">{{ _('Enable Auto-Approval') }}</button>
-        </li>
+        {% if addon.type in (amo.ADDON_EXTENSION, amo.ADDON_LPAPP) %}
```

Comment:
Observation: we should probably standardize this check (e.g. in a constant of addon types, or a utility function) because we check if auto approval is applicable in a few places and I'd bet with subtly different criteria in each.

Reply:
Yeah... I wanted to do that but the problem is, the 2 other places doing this check all need slightly different things:
- The `auto_approve` command itself filters a queryset, so we can't reuse that.
- The review page uses `is_ready_for_auto_approval()` to figure out whether or not it should display the result of auto-approval attempts on a not-yet public version. On top of the addon type, it checks the status (both of the add-on and the version).